core/src/trezor/lvglui/i18n/locales/ru.py

```
@@ -1042,11 +1042,14 @@
     "Количество слов в вашей seed-фразе не влияет на её безопасность — все они криптографически защищены.",
     "Показать QR-код",
     "BTC, TRON, SOL, ETH, BNB ...",
-    "Некоторые криптоактивы и аппаратные функции недоступны в режиме связи через QR-код. \nЭтот режим предназначен только для небольшого числа пользователей, которые редко пользуются своим аппаратным кошельком, и он несовместим с другими методами подключения. \nЕсли вы хотите подключить аппаратный кошелёк через Bluetooth или USB, пожалуйста, добавьте кошелёк заново, чтобы переключить режим связи.",
+    "Некоторые криптоактивы и аппаратные функции недоступны в режиме связи через QR-код. \n\nЭтот режим предназначен только для небольшого числа пользователей, которые редко пользуются своим аппаратным кошельком, и он несовместим с другими методами подключения. \n\nЕсли вы хотите подключить аппаратный кошелёк через Bluetooth или USB, пожалуйста, добавьте кошелёк заново, чтобы переключить режим связи.",
```

Ruff (0.14.7)

String contains ambiguous `о` (CYRILLIC SMALL LETTER O). Did you mean `o` (LATIN SMALL LETTER O)?

(RUF001)